### DISCUSSION TITLE
Is it possible to open local sqlite .db files in redash sqlite query runner??

### DISCUSSION THREAD

**Discussion by ashutosh6500** [getredash/redash#6350] [Q&A] [answered]
Is it possible to open local sqlite files in redash? If yes, how? Under what path should I place those files ?

**Comment by ashutosh6500** [comment, marked as answer]
One can place files in redash/redash folder and its present in container as /app/redash. Thus, for dbpath, /app/redash/filename should be given. In case of custom local directory, one can add volume in docker-compose file!

**Reply by justinclift** [reply]
Good info, that should be helpful for future people @ashutosh6500. :smile:
